include/fmt/format.h

```diff
@@ -71,6 +71,11 @@
 #  define FMT_HAS_BUILTIN(x) 0
 #endif
 
+#ifdef _MSC_VER
+#  pragma warning(push)
+#  pragma warning(disable : 4127)  // conditional expression is constant
```

Comment:
I'd rather avoid suppressing the warnings globally. It's better to fix/suppress in specific places because a warning may point to a genuine issue.

Reply:
This suppress in `format.h` was made similar to `format-inl.h`

Reply:
I'll look if this could be made more specific

Reply:
Fixed